src/Common/src/TypeSystem/IL/ILImporter.cs

```diff
@@ -304,13 +313,16 @@ private void MarkBasicBlock(BasicBlock basicBlock)
 
         private void ImportBasicBlock(BasicBlock basicBlock)
         {
-            _stackTop = 0;
+            _stack.Clear();
```

Comment:
Thanks for the clarification. I've moved the code over to ILToCppImporter. I've removed the WIP part in the title.